.gitignore

```diff
@@ -3,4 +3,4 @@ src/bidiMapper/.build/
 tests/__pycache__/
 package-lock.json
 src/.build/
-log.txt
+log.txt
```

Comment:
Please configure your editor to add trailing newlines at EOF. It reduces unnecessary line changes in future commits and avoids Git warnings. In VS Code: `"files.insertFinalNewline": true`

Reply:
done